endpoints.yml

```diff
@@ -0,0 +1,2100 @@
+---
+endpoints:
+  core:home:
+    - /core/
+  core:about:
+    - /core/about/
```

Comment:
For reproducibility and ease of comparison, it would be good to alpha-sort the endpoints by "name", I think?